components/Badges/Badges.css

```diff
@@ -2,17 +2,19 @@
     height: 1rem;
     width: 1rem;
     border-radius: var(--border-radius-f);
-    position: relative;
-   bottom: 1.75rem;
-   left: 3.75rem;
    border: 1px solid white;
+   
 }
 .red-badge{
     background-color: red;
+    margin-left: -1.5rem;
+    margin-top: 3.5rem;
 }
 
 .green-badge{
    background-color: green;
```

Comment:
here Instead of using color directly, define them in root first and then use it. It will be easy to change color next time if needed, you will have to just change the color in root and it will reflect everywhere.